crates/adapters/src/test/data.rs

```diff
@@ -326,6 +339,31 @@ impl TestStruct2 {
         ]))
     }
 
+    pub fn avro_schema() -> &'static str {
```

Comment:
the parquet test code has a struct with all the types we support.. maybe we should use that for tests

Reply:
we do :)

Reply:
i moved it here